auslib/web/admin/swagger/api.yaml

```diff
@@ -36,129 +50,418 @@ tags:
       url: "http://mozilla-balrog.readthedocs.io/en/latest/database.html#users"
 
   - name: "CSRF Token"
-    description: "CSRF Token Description"
+    description: "Cross-site request forgery Token for preventing execution of unauthorized commands"
     externalDocs:
       description: "Find out more about CSRF-Token"
       url: "http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#csrf-token"
 
-# basePath: "/api"
-# host: "localhost:8080"
-# Swagger-UI bug : fix TODO week 13
+  - name: "Releases"
+    description: "Balrog Releases Description"
+    externalDocs:
+      description: "Find out more about Releases"
+      url: "http://mozilla-balrog.readthedocs.io/en/latest/database.html#releases"
+
+parameters:
+  usernameParam:
+    name: username
+    in: path
+    description: unique identifier name of the user
+    type: string
+    maxLength: 100
+    required: true
+
+  csrfTokenParam:
+    name: csrf_token
+    in: query
+    description: valid csrf token
+    type: string
+    required: true
+
+  dataVersionParam:
+    name: data_version
+    in: query
+    description: data version of object.
+    type: integer
+    minimum: 1
+    required: true
+
+  roleParam:
+    name: role
+    in: path
```

Comment:
body parameters are defined in the definitions section as data models. Path and query parameters are defined in the parameters section. I created roleParam specifically to prevent defining it redundantly multiple times in the multiple user api path parameter. For signoffs, I'll probably define a new model or extend existing one when I start working on that.